### ISSUE TITLE
react-native 0.69.1 - "dependency.assets" is not allowed

### ISSUE THREAD

**Issue by megacherry** [adaptyteam/AdaptySDK-React-Native#26]
I've updated react-native to 0.69.1 and after that the pod is removed. if i build the app i get this error:

warn Package react-native-adapty has been ignored because it contains invalid configuration. Reason: "dependency.assets" is not allowed

```
diff --git a/node_modules/react-native-adapty/react-native.config.js b/node_modules/react-native-adapty/react-native.config.js
index f807944..f023cfe 100644
--- a/node_modules/react-native-adapty/react-native.config.js
+++ b/node_modules/react-native-adapty/react-native.config.js
@@ -6,7 +6,6 @@ module.exports = {
         sourceDir: './lib/android/',
       },
     },
-    assets: [],
   },
   project: {
     ios: {
```

**Comment by getinnocuous** [comment]
Also getting this error, any sign of a fix?

**Comment by megacherry** [comment]
Yeah, check out my first post. You need to edit the local package.json of react-native-adapty and then delete the line with "assets: []". Then run patch-package, that should fix it.

**Comment by megacherry** [comment]
please, can you fix it, it's a really easy fix. thanks ;)

**Comment by divanc** [comment]
Sorry for a late release! Fixed in v1.4
